content/en/docs/tasks/network/reconfigure-service-ip-ranges.md

```diff
@@ -0,0 +1,141 @@
+---
+reviewers:
+- thockin
+- dwinship
+min-kubernetes-server-version: v1.33
+title: Kubernetes Service CIDR Reconfiguration
+content_type: task
+---
+
+<!-- overview -->
+{{< feature-state feature_gate_name="MultiCIDRServiceAllocator" >}}
+
+This document shares how to reconfigure the existing Service IP range assigned
+to a cluster.
+
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+{{< version-check >}}
+
+<!-- steps -->
+
+## Kubernetes Service CIDR Reconfiguration
+
+This document focuses on the dynamic reconfiguration of the Kubernetes
+Service IP address range, a capability introduced by KEP-1880. Reconfiguring
```

Comment:
(Do we talk about KEPs in the end-user documentation? Does referring to KEP-1880 actually help the user here?)

Reply:
yeah, I should refer to the feature better